### DISCUSSION TITLE
Scikit-fda package

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#149] [BaiduKDDCup2022] [answered]
Please, install scikit-fda  in base environment
pip install scikit-fda

Thank in advanced

**Comment by ramber1836** [comment, marked as answer]
Could you explain the necessity of using this pkg?

**Reply by fersebas** [reply]
Use smooth techniques for data :)

**Reply by ramber1836** [reply]
The package has already been added to the base env.
